x-pack/solutions/search/plugins/search_playground/common/index.ts

```diff
@@ -21,3 +21,4 @@ export const DEFAULT_PAGINATION: Pagination = {
   size: 10,
   total: 0,
 };
+export const ContextModelLimitError = 'Context exceeds the model limit';
```

Comment:
shouldn't this be translated?

Reply:
currently this value is used in throwing [error](https://github.com/elastic/kibana/blob/2590ffef38e87950d7a462adc94a93e203d4d625/x-pack/solutions/search/plugins/search_playground/server/lib/conversational_chain.ts#L107) & in system message comparison to update system [context limit length error message](https://github.com/elastic/kibana/blob/2590ffef38e87950d7a462adc94a93e203d4d625/x-pack/solutions/search/plugins/search_playground/public/components/message_list/system_message.tsx#L34C21-L34C43). I wonder if we need translation for this use case?

Reply:
I think we do yes, especially in this case: [context limit length error message](https://github.com/elastic/kibana/blob/2590ffef38e87950d7a462adc94a93e203d4d625/x-pack/solutions/search/plugins/search_playground/public/components/message_list/system_message.tsx#L34C21-L34C43) we would be templating in an english error message into a translated error message. I think they should both be translated so that the languages match.